packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -322,6 +325,21 @@ class Splitter extends (CollectionWidget as any) {
     });
   }
 
+  _updateNestedSplitterOption(optionName: string, optionValue: unknown): void {
+    this._iterateItems((index, itemElement) => {
```

Comment:
I think that we should only influence splitters that are declared via items[].splitter. In other cases, templates are used, and we don't know the customers' business cases there

So, the draft version of the code could be as follows
```
this.option('items').forEach((item) => {
      if (item?.splitter) {
        getComponentInstance($(this._getElementByItem(item)).find('.dx-splitter').eq(0)).option(optionName, optionValue);
      }
    });
```
The _getElementByItem method is already used in the menu, and we can either duplicate this or move it to a common parent level